CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### 0.19.0
+
+* Fixed the binding for `Webapi.Url.URLSearchParams.entries` to return `(string, string)`
+
 ### 0.18.1
 
 * Added `Webapi.Dom.Node.replaceChild`
```

Comment:
Can you fold this log entry into 0.19.0, and get rid of the 0.18.1 release here, thx